lib/ruby_lexer.rb

```diff
@@ -146,10 +147,11 @@ def expr_result token, text
   def heredoc here # TODO: rewrite / remove
     _, eos, func, last_line = here
 
-    indent  = (func & STR_FUNC_INDENT) != 0 ? "[ \t]*" : nil
-    expand  = (func & STR_FUNC_EXPAND) != 0
-    eos_re  = /#{indent}#{Regexp.escape eos}(\r*\n|\z)/
-    err_msg = "can't match #{eos_re.inspect} anywhere in "
+    indent         = (func & STR_FUNC_INDENT) != 0 ? "[ \t]*" : nil
+    content_indent = (func & STR_FUNC_ICNTNT) != 0
+    expand         = (func & STR_FUNC_EXPAND) != 0
+    eos_re         = /#{indent}#{Regexp.escape eos}(\r*\n|\z)/
+    err_msg        = "can't match #{eos_re.inspect} anywhere in "
```

Comment:
If you're going to add a new item that changes alignment, do it in two commits... one where the reviewer can look at content changes, and the other that can be glossed over as reformatting. Your single commit means I need to mentally refine the diff.